packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -326,9 +326,32 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
     const region = Lazy.uncachedString({
       produce: (context) => Stack.of(context.scope).region,
     });
-    this.name = `${prefix || 'com.amazonaws'}.${region}.${name}`;
+    const regionPrefix =Lazy.uncachedString({
```

Comment:
```suggestion
    const defaultEndpointPrefix =Lazy.uncachedString({
```
How about just call it `defaultEndpointPrefix` ?

Reply:
Thanks @Pahud, that's a good suggestion, I'll update it.